content/en/guide/v10/differences-to-react.md

```diff
@@ -12,7 +12,7 @@ The reason Preact does not attempt to include every single feature of React is i
 
 ---
 
-<div><toc></toc></div>
+<toc></toc>
```

Comment:
Carful here, we wrapped the `<toc>` with a `<div>` for a reason. Our markdown parser treats any custom-element as an inline element and wraps them with a `p` tag. This leads to invalid HTML here which breaks SSR. See https://github.com/preactjs/preact-www/pull/573 for the PR.

Reply:
nvm, this was changed as soon as I made the comment :joy:

Reply:
lol I 100% botched this edit and submitted a PR to a fork that was apparently behind master 🤪

I _think_ I got everything rebased manually - if there's anything that looks off lmk.